base/alltt.dtx

```diff
@@ -104,7 +104,7 @@
 %  The glyph at the position of the $ in an italic font: \textit{$}.
 %    \end{alltt}
 %
-%  \StopEventually{}
+%  \MaybeStopHere{}
```

Comment:
I've always thought it's more (literally) `\StopPrehaps` or more fluidly `\ConditionalStop`/`\PotentialStop`

Reply:
@u-fischer  suggests `\MaybeStop` :)

Reply:
fine by me